pkg/apis/cluster/v1alpha1/types.go

```diff
@@ -278,6 +278,9 @@ type LocalSecretReference struct {
 const (
 	// ClusterConditionReady means the cluster is healthy and ready to accept workloads.
 	ClusterConditionReady = "Ready"
+
+	// ClusterConditionCompleteAPIEnablements means the cluster's APIEnablements are complete.
+	ClusterConditionCompleteAPIEnablements = "CompleteAPIEnablements"
```

Comment:
Do we need to add the condition to file https://github.com/karmada-io/karmada/blob/70afc1ad6d170bd4773b5e7a9735a4116816f814/pkg/apis/cluster/types.go?

@XiShanYongYe-Chang

Reply:
Although there is no logic to call this constant, I recommend adding it to this file, consistent with the previous ClusterConditionReady.